### ISSUE TITLE
Too much space above and below reduced size fractions

### ISSUE THREAD

**Issue by John02139** [stipub/stixfonts#127]
```
\documentclass[11pt]{article}
\usepackage{unicode-math}
\setmainfont{STIX Two Text}
\setmathfont{STIX Two Math}

\begin{document}

\[  
\frac{\cos x}{\sin x} = \frac{1}{\frac{\sin x}{\cos x}}
\]

\end{document}
```

<img width="284" alt="screen shot 2018-09-09 at 3 11 07 pm" src="https://user-images.githubusercontent.com/39748301/45267884-9d051f80-b442-11e8-9708-838827e3b0a0.png">

The separation between sin x, cos x, and the lower horizontal rule in the fraction at right is larger than in other fonts -- does not seem to be scaling with the font size

**Comment by marcianx** [comment]
Agreed. In particular, the extra space is _really_ jarring in `\textstyle`

E.g. on XeTeX with
```
$$
{\scriptscriptstyle\frac{x}{2}}
{\scriptstyle\frac{x}{2}}
{\textstyle\frac{x}{2}}
{\displaystyle{\frac{x}{2}}}
{\scriptscriptstyle {x \atop 2}}
{\scriptstyle {x \atop 2}}
{\textstyle {x \atop 2}}
{\displaystyle {x \atop 2}}
$$
```
I get
![image](https://user-images.githubusercontent.com/370338/67530467-24384580-f68d-11e9-8409-12080eaba47c.png)
(especially the third one)
whereas the regular LaTeX font gives something far more compact:
![image](https://user-images.githubusercontent.com/370338/67530637-b6d8e480-f68d-11e9-92c9-6c4da8c95e78.png)

**Comment by tiroj** [comment]
@davidmjones Is issue still live? And is it something specific to XeTex or is there something we can do in the font to affect this?

**Comment by yjhn** [comment]
Same issue with appears with LuaTeX.

**Comment by csalinezh** [comment]
A quick non-exhaustive search shows multiple complaints and workarounds using LuaTeX's `\Umathfractionnumup`, `\Umathfractiondenomdown` or other hacky ways to patch this in-line fraction behaviour:
https://tex.stackexchange.com/questions/387918/spacing-between-bars-in-textstyle-fractions-issue-with-stix2-or-opentype-math-s
https://tex.stackexchange.com/questions/622745/vertical-spacing-in-text-style-fractions-with-stix-2-fonts
https://tex.stackexchange.com/questions/426491/open-type-math-font-stix-letters-are-very-close-to-each-others-in-some-places
https://tex.stackexchange.com/questions/385715/how-to-make-persistent-changes-to-math-parameters
